apps/website/inertia/pages/events/show/page.tsx

```diff
@@ -289,6 +296,16 @@ export default function EventRegistrationPage({
                   <p className="text-lg font-semibold">Requisitos de Inscrição</p>
                 </h1>
               )}
+              {/* Extra Information */}
+              {extraInfo && (
+                <div>
+                  <h1 className="mb-3 flex items-center gap-2 text-lg font-semibold">
+                    <Info className="h-5 w-5" />
+                    <p className="text-lg font-semibold">Informação Extra</p>
+                  </h1>
+                  <div dangerouslySetInnerHTML={{ __html: extraInfo }} />
```

Comment:
I know you said that we can change this content through the admin panel but is there some other kind of validation on this besides our privileged access?

Reply:
probably not, I think we just have to deal with it, this is not meant to be writable outside of the admin panel